src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,14 +16,35 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
-    public Integer getWidth() {
-        return width;
+    @Override
+    public int hashCode() {
+        int result = 0;
```

Comment:
```suggestion
        int result = 31;
```
we should use some primary number)